media/js/firefox/lockwise/lockwise.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+
+    if (client._isFirefox() === true && client._getFirefoxMajorVersion() >= '70') {
+        document.querySelector('.for-firefox-69-and-below').remove();
+        document.querySelector('.for-non-firefox-users').remove();  
+    }
+    else if ( client._isFirefox() === true && client._getFirefoxMajorVersion() < '70' ) {
+        document.querySelector('.for-firefox-70-and-above').remove();
+        document.querySelector('.for-non-firefox-users').remove();
+    }
+    else if ( client._isFirefox() === false ){
+        document.querySelector('.for-firefox-69-and-below').remove();
+        document.querySelector('.for-firefox-70-and-above').remove();
+    } else {
+        document.querySelector('.for-firefox-70-and-above').remove();
+        document.querySelector('.for-firefox-69-and-below').remove();
+        document.querySelector('.for-non-firefox-users').remove();
+    }
+
+    document.querySelector('#lockwise-button').addEventListener('click', function() {
```

Comment:
We should probably only add this event listener if the visitor is on firefox 70+.

In Chrome I see `Uncaught TypeError: Cannot read property 'addEventListener' of null`